installer/roles/kubernetes/defaults/main.yml

```
@@ -40,6 +40,7 @@ openshift_pg_emptydir: false
 openshift_pg_pvc_name: postgresql
 
 kubernetes_deployment_name: awx
+kubernetes_serviceaccount_name: "{{ kubernetes_deployment_name }}"
```

```suggestion
kubernetes_serviceaccount_name: awx
```

This could be a breaking change for users for those already using a non-default `kubernetes_deployment_name`. Can set the default value to 'awx` instead?

---

Sure, I hadn't thought about that
It's pushed